pkg/http/listener.go

```diff
@@ -89,13 +90,22 @@ type httpListener struct {
 	errorLogFunc           func(error, string, ...interface{}) // function to be called on errors.
 }
 
-// ignoreErr returns true if error is due to a network timeout or
+// isRoutineNetErr returns true if error is due to a network timeout or
 // io.EOF and false otherwise
-func ignoreErr(err error) bool {
+func isRoutineNetErr(err error) bool {
```

Comment:
I think you can retain the function name as `ignoreErr()`. It is clear enough I guess.

Reply:
Done